README.md

```diff
@@ -1 +1,53 @@
-# To-Do-list
\ No newline at end of file
+# To-Do-list
+
+
+> In this app I made list of To Do tasks which is not interactive at this time. This web page is built using webpack.
+
+## Built With
+
+- HTML and CSS
+- JavaScript
+- webpack
+- VS studio code, GitHub.
+
+## Getting Started
+
+**You can run the application locally by cloning this [repository](https://github.com/atatm/To-Do-list.git)**
+
+### Local Setup
+
+To run this project locally, make sure you have done the following:
+
+- Created hint.rc file
+- Created stylelint.rc file
+- Created .gihub/workflows/linters/yml
+- Get a copy of the project by running `https://github.com/atatm/To-Do-list.git `
+- cd into the project
+- Open the project using code editor of your choice
+- Open your prefered browser
+
+## Authors
+
+**Alelign Ayana**
+
+- LinkedIn: [Alelign Ayana](https://linkedin.com/@alelignayana)
+
+## :handshake: Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+### Go Live
+
+https://atatm.github.io/To-Do-list/
```

Comment:
- [ ] Kindly add `/dist/index.html` to this link so it becomes a live demo to the app, right now it's taking the users to your README file 👍

Reply:
Fixed.